styles.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
```

Comment:
- [ ] Please consider using the same `color, padding, margin, font size`, and all the rest as per the original design, take a look at this below compared to your design 
- [ YOUR DESIGN]

<img width="310" alt="luca" src="https://user-images.githubusercontent.com/79058364/159028307-15c7caec-149a-4360-b248-e3d8e0ec831e.png">
 

- [ ] Do all necessary changes to match this design below ⬇️ 

- [ FIGMA]
<img width="256" alt="BAK" src="https://user-images.githubusercontent.com/79058364/159028568-4c195e98-3576-45f3-8176-667cf86919dc.png">